### ISSUE TITLE
Page Structure Upgrade for ADA

### ISSUE THREAD

**Issue by kzadrozny** [uchicago-library/universalviewer#3]
Viewing code from [our instance](https://iiif-viewer.lib.uchicago.edu/uv/uv.html#?manifest=http://iiif-manifest.lib.uchicago.edu/mvol/0004/1903/0302/mvol-0004-1903-0302.json) of the UniversalViewer.
_Details on specific fixes listed after summary._
- [ ] Missing page title
- [ ] No landmarks / page regions

### Suggested Code
```
<head>
    <title>[Collection Title]</title>
...
</head>
```

## No landmarks / page regions
Documentation 1: https://www.w3.org/WAI/GL/wiki/Using_ARIA_landmarks_to_identify_regions_of_a_page
Documentation 2: https://www.w3.org/WAI/tutorials/page-structure/regions/

### Suggested Code
**Very high-level overview**
```
<div id="headerPanel" role="navigation" aria-label="Primary"> [ previous / next buttons ] </div>
<div id="leftPanel" role="navigation" aria-label="Secondary"> [ object thumbnails ] </div>
<div id="content" role="main"> [ object image ] </div>
<div id="rightPanel" role="complementary"> [ object metadata ] </div>
<div id="footer" role="form" aria-label="Image Options"> [ footer items ] </div>
```

**Comment by verbalhanglider** [comment]
See https://iiif-viewer.lib.uchicago.edu/beta/uv/uv.html#?manifest=https%3A%2F%2Fiiif-manifest.lib.uchicago.edu%2Fmvol%2F0004%2F1924%2F1203%2Fmvol-0004-1924-1203.json&c=0&m=0&s=0&cv=0&xywh=-6092611%2C-421%2C12190999%2C8349

I have added a title element to the page.
